### PR TITLE
colorz: init at 1.0.3

### PR DESCRIPTION
**Pull request by skykanin**
###### Motivation for this change
This PR introduces colorz a python based tool for generation color schemes from images

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -6620,6 +6620,12 @@
     email = "jot.skrzyp@gmail.com";
     name = "Jakub Skrzypnik";
   };
+  skykanin = {
+    email = "skykanin@users.noreply.github.com";
+    github = "skykanin";
+    githubId = 3789764;
+    name = "skykanin";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]: should be a separate commit:
```
maintainers: add skykanin
```

File: pkgs/tools/misc/colorz/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0ghd90lgplf051fs5n5bb42zffd3fqpgzkbv6bhjw7r8jqwgcky0";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ pillow scipy ];
+
+  meta = with lib; {
+    description = "Color scheme generator";
+    homepage = https://github.com/metakirby5/colorz;
```
Comment (jonringer) [inline pkgs/tools/misc/colorz/default.nix]:
```suggestion
    homepage = "https://github.com/metakirby5/colorz";
```

**Review by jonringer — COMMENTED** [review]
should just be 2 commits:
```
maintainers: add skykanin
colorz: init at 1.0.3
```

File: pkgs/tools/misc/colorz/default.nix (hunk 2 above)
Comment (skykanin) [reply pkgs/tools/misc/colorz/default.nix]: oh, ok. I didn't use strings for the homepage url since this isn't done in examples from th e nixpkgs manual either https://nixos.org/nixpkgs/manual/#chap-meta

Comment (jonringer) [reply pkgs/tools/misc/colorz/default.nix]: oh, we need to change that, an rfc was merged recent

File: pkgs/tools/misc/colorz/default.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  pname = "colorz";
+  version = "1.0.3";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0ghd90lgplf051fs5n5bb42zffd3fqpgzkbv6bhjw7r8jqwgcky0";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ pillow scipy ];
+
```
Comment (jonringer) [inline pkgs/tools/misc/colorz/default.nix]: be nice to have some sanity check that pillow and scipy are imported correctly
```suggestion
  checkPhase = ''
    $out/bin/colorz --help > /dev/null
  '';
```

**Comment by jonringer** [comment]
should just be 2 commits:

```
maintainers: add skykanin
colorz: init at 1.0.3
```
https://nixos.org/nixpkgs/manual/#submitting-changes-making-patches

**Review by jonringer — APPROVED** [review]
LGTM
application shows usage

```
https://github.com/NixOS/nixpkgs/pull/80188
1 package built:
colorz
```
